src/test/java/rv/atqc/gttesting/archex/AbstractPage.java

```diff
@@ -62,4 +62,12 @@ public T waitForInVisibilityOfAll(ArrayList<WebElement> elements, int timeout) {
 		 wait.until(ExpectedConditions.invisibilityOfAllElements(elements));
 		 return (T) this;
 	}
+
+    public void placeCursorOverElement(WebElement element){
+        String javaScript = "var evObj = document.createEvent('MouseEvents');" +
```

Comment:
if you need just to move the cursor over the element, you can use Actions:

Actions builder = new Actions(driver);
builder.moveToElement(element).perform();